testsuite/tests/typing-objects/Exemples.ml

```diff
@@ -542,10 +542,7 @@ Error: Type
            < cmp : int_comparable2 -> int; set_x : int -> unit; x : int >
        is not a subtype of
          int_comparable = < cmp : int_comparable -> int; x : int >
-       Type int_comparable = < cmp : int_comparable -> int; x : int >
-       is not a subtype of
-         int_comparable2 =
-           < cmp : int_comparable2 -> int; set_x : int -> unit; x : int >
+       Type int_comparable is not a subtype of int_comparable2
```

Comment:
I think this change is not deliberate, but not a bug. There is a frankly insane part of the type printer that decides whether to print more details about an error by marshalling the types to see how big they are. Since we now have a `Tpoly` node in our parameter types, these sizes are slightly bigger for function types, and so some errors will now print less detail.